src/index.ts

```diff
@@ -251,33 +246,33 @@ export async function install(installTargets: InstallTarget[], installOptions: I
 
   const inputOptions = {
     input: depObject,
-    external: externalPackages,
+    external: externalPackage,
```

Comment:
this comment applies here, and across the rest of the function:

Can we keep the existing names in the function by doing something like this:

```js
installOptions: {
      externalPackage: externalPackages,
      strict: isStrict,
      babel: isBabel,
      // ... etc
    },
  } = config;
```

I know that probably feels like a pain to maintain, but it's much better than using confusing variable names in the code itself. Like how `externalPackage` sounds like a single package, but is an array. `babel` sounds like the babel package, and not a boolean flag, etc.

You can also look into doing that transform when config is created instead, but that will be a bit more work and I'm honestly fine doing the transform here for now.